docs/trex_error_handling.md

```diff
@@ -44,3 +43,49 @@ tableau.extensions.ui.displayDialogAsync(args... ).then((args... ) => {
 
 ```
 
+## Handle Extensions API errors when the dashboard is not visible
+
+ In Tableau Server or Tableau Online version 2018.3 and later, when the browser window is not visible (that is, when the browser window Tableau is running in is minimized or in the background), the Extensions API method calls are blocked and an error object is returned. If you have code that might run when the dashboard is not visible, you should add code to check if the window is visible so that you can handle the error. If you are using `tableau-extensions-1.1.0.js` or later, the error code returned in this case is `VISIBILITY_ERROR`.
```

Comment:
I think the visibility error was turned off for in all the MRs now. you can check with sean for the exact versions and more details.

Reply:
Thanks, @jz-huang. For some reason this merge request is bringing in  content we merged in 1.1. No matter. I'll check with @seanmakesgames